azure-kusto-data/source/client.ts

```diff
@@ -98,14 +100,41 @@ export class KustoClient {
         return this._execute(endpoint, ExecutionType.Ingest, db, null, stream, properties);
     }
 
+    executeWithDefaultDatabase(query: string, properties?: ClientRequestProperties) {
```

Comment:
For this and all the other new methods - If we want to align with C# client we should introduce executeQuery(query: string, properties?: ClientRequestProperties) method.